common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js

```diff
@@ -0,0 +1,24 @@
+try {
+    ((document, window) => {
+        var ad = document.createElement('div');
+        ad.style.position = 'absolute';
+        ad.style.left = '-9999px';
+        ad.style.height = '10px';
+        ad.innerHTML = '&nbsp;';
+        ad.setAttribute('class', 'ad_unit');
+
+        window.requestAnimationFrame(() => {
+            document.body.appendChild(ad);
```

Comment:
I wonder if you really need any `rAF` call. since the script is inline at the top of the body element, the browser is anyway currently building the DOM, and so the render tree is itself either inexistant or incomplete.